### PR TITLE
qa/suites/rados/thrash-old-clients: Add noscrub, nodeep-scrub to ignorelist

### PR DESCRIPTION
**Pull request by kamoltat**
RADOS main suite has been failing due to
a lack of whitelisting for

`noscrub` and `nodeep-scrub`

Fixes: https://tracker.ceph.com/issues/65824

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [x] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [x] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [ ] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

File: qa/tasks/thrashosds-health.yaml [hunk 1]
```diff
@@ -28,3 +28,5 @@ overrides:
       - mons down
       - mon down
       - out of quorum
+      - noscrub
+      - nodeep-scrub
```
Comment (ljflores) [inline qa/tasks/thrashosds-health.yaml:32]: I would check what this red mark means, but otherwise LGTM

**Review by ljflores — APPROVED** [review]

File: qa/tasks/thrashosds-health.yaml (hunk 1 above)
Comment (kamoltat) [reply qa/tasks/thrashosds-health.yaml:32]: I think I need to add one more spacing

**Comment by kamoltat** [comment]
I just pushed a new commit, with an additional line, getting rid of the red mark discussed in comments above.

**Comment by kamoltat** [comment]
scheduled 7 jobs to https://pulpito.ceph.com/ksirivad-2024-07-01_16:30:09-rados-main-distro-default-smithi/

**Comment by kamoltat** [comment]
https://pulpito.ceph.com/ksirivad-2024-07-01_19:06:20-rados-main-distro-default-smithi/7781829/

**Comment by kamoltat** [comment]
Since this change only involves adding things to a test file and being exclusive to only thrashosd-health. We do not need to schedule a full rados test

**Pull request merged**